block-builder/src/main.rs

```diff
@@ -1,58 +1,30 @@
-use clap::{Parser, Subcommand};
```

Comment:
Lets remove `clap` from dependencies since it is not used

Reply:
Done eb8e325